owncloudApp/build.gradle

```diff
@@ -26,6 +26,7 @@ dependencies {
 
     // Image loading
     implementation 'com.github.bumptech.glide:glide:4.12.0'
+    implementation 'com.github.corouteam:GlideToVectorYou:v2.0.0'
```

Comment:
Is this dependency needed?

Reply:
The library is based on Glide and offers the same functionalities + SVG support. Just by adding the dependency, SVG support works. Maybe the question is... Do we need the original glide dependency if we use this new dependency or can they coexist together? What do you think?